browser-extension/mv2/src/modules/EventActions.js

```diff
@@ -186,3 +179,22 @@ EventActions.stopPeriodicEventWriter = () => {
     EventActions.eventWriterInterval = null;
   }
 };
+
+/* CONSUMER UTILITIES */
+EventActions.EVENTS = {
+  RULE_EXECUTED: "rule_executed",
+};
+
+EventActions.sendRuleExecutionEvent = (rule) => {
+  const eventParams = {
+    rule_type: rule.ruleType,
+    rule_id: rule.id,
+  };
+
+  eventParams["log_source"] = "extension";
```

Comment:
`source` is not present in `eventParams`

Reply:
@nafees87n @nsrCodes Instead of `source`, can we rather set `platform=extension`. Other values of platform - desktop, mobile, etc. 
On a single platform, there will always be one source. That is, `background` in `extension`, `proxy` in `desktop`, `sdk` in `mobile`. So, we can skip `source`. WDYT?

Reply:
@lazyvab as discussed earlier, IMO `source` sounded confusing as currently we use it for a very different context in web app events. I would like using `platform` instead. 

So just to be clear, `platform` would replace `log_source` and `source` would remain as is.